unthink-stack/src/server/framework/service-definition.ts

```diff
@@ -0,0 +1,17 @@
+import { ServiceResult } from './service-result';
+
+type HandlerMethod = 'GET' | 'PUT' | 'POST' | 'DELETE';
+
+export interface HandlerContext {
+  query?: Record<string, string>;
+  path?: Record<string, string>;
```

Comment:
```suggestion
  queryParams?: Record<string, string>;
  pathParams?: Record<string, string>;
```

Just a thought. In Angular, the `Route` information names them this way. To me, it clarifies what they are. `path` by itself does not sound like parameter information, but instead sounds like the URL.

Reply:
Yeah totally agree. I think that was copied from how express does it, which to your point is even confusing there when you see `path.